app/src/main/java/br/com/mgalhardo/guidebook/dagger/UiComponent.java

```diff
@@ -0,0 +1,12 @@
+package br.com.mgalhardo.guidebook.dagger;
+
+import br.com.mgalhardo.guidebook.dagger.module.presentation.PresenterModule;
+import br.com.mgalhardo.guidebook.dagger.scope.PerActivity;
+import br.com.mgalhardo.guidebook.presentation.ui.guidelist.GuideListActivity;
+import dagger.Subcomponent;
+
+@PerActivity
```

Comment:
@viniciussoares 
Sorry if I am missing something, but... Does this mean you will have a new instance of this subcomponent for every activity you inject it into? If this is the case, could you please explain why this is useful here as it is with presenters? Thanks!